test/performance/alphabet/alphabet_benchmark.cpp

```diff
@@ -89,52 +89,52 @@ static void to_rank(benchmark::State& state)
-BENCHMARK_TEMPLATE(to_char, dna5);
-BENCHMARK_TEMPLATE(to_char, dna15);
-BENCHMARK_TEMPLATE(to_char, rna15);
-BENCHMARK_TEMPLATE(to_char, rna4);
-BENCHMARK_TEMPLATE(to_char, rna5);
-BENCHMARK_TEMPLATE(to_char, char);
-BENCHMARK_TEMPLATE(to_char, gapped<dna4>);
-BENCHMARK_TEMPLATE(to_char, alphabet_variant<gap,dna4,dna5,dna15,rna15,rna4,rna5>);
-BENCHMARK_TEMPLATE(to_char, alphabet_variant<char, dna4, dna5, dna15>);
+BENCHMARK_TEMPLATE(to_char_, gap);
+BENCHMARK_TEMPLATE(to_char_, dna4);
+BENCHMARK_TEMPLATE(to_char_, dna5);
+BENCHMARK_TEMPLATE(to_char_, dna15);
+BENCHMARK_TEMPLATE(to_char_, rna15);
+BENCHMARK_TEMPLATE(to_char_, rna4);
+BENCHMARK_TEMPLATE(to_char_, rna5);
+BENCHMARK_TEMPLATE(to_char_, char);
+BENCHMARK_TEMPLATE(to_char_, gapped<dna4>);
+BENCHMARK_TEMPLATE(to_char_, alphabet_variant<gap,dna4,dna5,dna15,rna15,rna4,rna5>);
+BENCHMARK_TEMPLATE(to_char_, alphabet_variant<char, dna4, dna5, dna15>);
 
-BENCHMARK_TEMPLATE(assign_rank, gap);
-BENCHMARK_TEMPLATE(assign_rank, dna4);
-BENCHMARK_TEMPLATE(assign_rank, dna5);
-BENCHMARK_TEMPLATE(assign_rank, dna15);
-BENCHMARK_TEMPLATE(assign_rank, rna15);
-BENCHMARK_TEMPLATE(assign_rank, rna4);
-BENCHMARK_TEMPLATE(assign_rank, rna5);
-BENCHMARK_TEMPLATE(assign_rank, char);
-BENCHMARK_TEMPLATE(assign_rank, gapped<dna4>);
-BENCHMARK_TEMPLATE(assign_rank, alphabet_variant<gap,dna4,dna5,dna15,rna15,rna4,rna5>);
-BENCHMARK_TEMPLATE(assign_rank, alphabet_variant<char, dna4, dna5, dna15>);
+BENCHMARK_TEMPLATE(assign_rank_, gap);
+BENCHMARK_TEMPLATE(assign_rank_, dna4);
+BENCHMARK_TEMPLATE(assign_rank_, dna5);
+BENCHMARK_TEMPLATE(assign_rank_, dna15);
+BENCHMARK_TEMPLATE(assign_rank_, rna15);
+BENCHMARK_TEMPLATE(assign_rank_, rna4);
+BENCHMARK_TEMPLATE(assign_rank_, rna5);
+BENCHMARK_TEMPLATE(assign_rank_, char);
+BENCHMARK_TEMPLATE(assign_rank_, gapped<dna4>);
+BENCHMARK_TEMPLATE(assign_rank_, alphabet_variant<gap,dna4,dna5,dna15,rna15,rna4,rna5>);
+BENCHMARK_TEMPLATE(assign_rank_, alphabet_variant<char, dna4, dna5, dna15>);
 
-BENCHMARK_TEMPLATE(to_rank, gap);
-BENCHMARK_TEMPLATE(to_rank, dna4);
-BENCHMARK_TEMPLATE(to_rank, dna5);
-BENCHMARK_TEMPLATE(to_rank, dna15);
-BENCHMARK_TEMPLATE(to_rank, rna15);
-BENCHMARK_TEMPLATE(to_rank, rna4);
-BENCHMARK_TEMPLATE(to_rank, rna5);
-BENCHMARK_TEMPLATE(to_rank, char);
-BENCHMARK_TEMPLATE(to_rank, gapped<dna4>);
-BENCHMARK_TEMPLATE(to_rank, alphabet_variant<gap,dna4,dna5,dna15,rna15,rna4,rna5>);
-BENCHMARK_TEMPLATE(to_rank, alphabet_variant<char, dna4, dna5, dna15>);
+BENCHMARK_TEMPLATE(to_rank_, gap);
+BENCHMARK_TEMPLATE(to_rank_, dna4);
+BENCHMARK_TEMPLATE(to_rank_, dna5);
+BENCHMARK_TEMPLATE(to_rank_, dna15);
+BENCHMARK_TEMPLATE(to_rank_, rna15);
+BENCHMARK_TEMPLATE(to_rank_, rna4);
+BENCHMARK_TEMPLATE(to_rank_, rna5);
+BENCHMARK_TEMPLATE(to_rank_, char);
+BENCHMARK_TEMPLATE(to_rank_, gapped<dna4>);
+BENCHMARK_TEMPLATE(to_rank_, alphabet_variant<gap,dna4,dna5,dna15,rna15,rna4,rna5>);
+BENCHMARK_TEMPLATE(to_rank_, alphabet_variant<char, dna4, dna5, dna15>);
```

Comment:
So how are the benchmarks? same/faster/slower?

Reply:
I checked, it's the same. I mean, everything else would be strange, it just calls the same functions in the end.